app/src/lib/stores/app-store.ts

```diff
@@ -1025,20 +1025,23 @@ export class AppStore extends TypedBaseStore<IAppState> {
         ? cachedDefaultBranch
         : null
 
+    const aheadBehindUpdater = this.currentAheadBehindUpdater
     let inferredBranch: Branch | null = null
     let aheadBehindOfInferredBranch: IAheadBehind | null = null
-    if (tip.kind === TipState.Valid && compareState.aheadBehindCache !== null) {
+    if (
+      tip.kind === TipState.Valid &&
+      compareState.aheadBehindCache !== null &&
+      aheadBehindUpdater !== null
```

Comment:
this is a nice consolidation of logic! no need to find the inferred branch if we're not going to do anything with it